collect_app/src/main/res/layout/storage_migration_dialog.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="?attr/colorSurface"
+    android:orientation="vertical">
+
+    <include layout="@layout/material_full_screen_dialog_toolbar" />
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical"
+            android:padding="@dimen/margin_standard">
+
+            <TextView
+                android:id="@+id/messageText1"
+                style="@style/TextAppearance.Collect.Body1"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/margin_small"
+                android:text="@string/storage_migration_dialog_message1" />
+
+            <TextView
+                android:id="@+id/messageText2"
+                style="@style/TextAppearance.Collect.Body1"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:visibility="gone"
+                android:layout_marginTop="@dimen/margin_small"
+                android:text="@string/storage_migration_dialog_message2" />
+
+            <TextView
+                android:id="@+id/messageText3"
+                style="@style/TextAppearance.Collect.Body1"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/margin_small"
+                android:text="@string/storage_migration_dialog_message3" />
+
+            <TextView
+                android:id="@+id/moreDetailsButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/storage_migration_more_details"
+                android:textColor="?colorAccent"
```

Comment:
Again we should be using `colorSecondary` instead of `colorAccent`

Reply:
Done.